src/Storages/System/StorageSystemTables.cpp

```diff
@@ -239,6 +246,38 @@ class TablesBlockSource : public ISource
     String getName() const override { return "Tables"; }
 
 protected:
+    NameToNameMap getSelectParamters(const StorageMetadataPtr & metadata_snapshot)
+    {
+        const SelectQueryDescription & query_description = metadata_snapshot->getSelectQuery();
+        ASTPtr inner_query = query_description.inner_query;
+        if (!inner_query || !inner_query->as<ASTSelectWithUnionQuery>())
+            return {};
+
+        return inner_query->as<ASTSelectWithUnionQuery>()->getQueryParameters();
+    }
+
+    void fillParametralizedViewData(MutableColumns & columns, const StoragePtr & table, size_t & res_index, size_t & src_index)
+    {
+        if (src_index == 14 && columns_mask[src_index++])
```

Comment:
```suggestion
        if (src_index == 14 && columns_mask[src_index])
        {
         ...
         src_index++;
        }
```

to fix the clang tidy build

Reply:
Looks like src_index++ is not needed, its already done in the while loop where this function is called. so just 

```suggestion
        if (src_index == 14 && columns_mask[src_index])
```

Reply:
This is for temporary table.

There is another place which fillParametralizedViewData is called for normal table. I think that this condition should be removed from this function and checked before the function is called in this case.